dpnp/dpnp_iface_sorting.py

```diff
@@ -59,9 +59,9 @@ def argsort(in_array1, axis=-1, kind=None, order=None):
     -----------
     Input array is supported as :obj:`dpnp.ndarray`.
     Otherwise the function will be executed sequentially on CPU.
-    Prameters ``axis`` is supported only with default value ``-1``.
-    Prameters ``kind`` is supported only with default value ``None``.
-    Prameters ``order`` is supported only with default value ``None``.
+    Parameters ``axis`` is supported only with default value ``-1``.
```

Comment:
If you want, you can make such a modification in this task as well.

```suggestion
    Parameters `axis` is supported only with default value ``-1``.
```